proctor-common/src/main/java/com/indeed/proctor/common/model/PayloadExperimentConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.indeed.proctor.common.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Builder
+public class PayloadExperimentConfig {
+    @Nullable private String priority;
+    @Nonnull private List<String> namespaces = Collections.emptyList();
+
+    @Override
+    public String toString() {
```

Comment:
Does the `toString()` generated by `@Data` mangle `namespaces`?

Reply:
No the issue is that is generates using `()` isntead `{}`.  Had to override to match style from Test Definition toString() which uses {}